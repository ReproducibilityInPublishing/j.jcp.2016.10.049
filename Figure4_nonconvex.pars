2	39
2	79
2	119
2	159
2	199
2	239
1	39
1	79
1	119
1	159
1	199
1	239
