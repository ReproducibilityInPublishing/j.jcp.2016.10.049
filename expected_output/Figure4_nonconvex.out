"Lattice Size", "NY", "Tau", "Error"
0.02500000000000, 39, 2.00000000000000, 0.02313219157371
0.01250000000000, 79, 2.00000000000000, 0.00635629001330
0.00833333333333, 119, 2.00000000000000, 0.00269025891897
0.02500000000000, 39, 1.00000000000000, 0.00832190276227
0.01250000000000, 79, 1.00000000000000, 0.00198378311328
0.00625000000000, 159, 2.00000000000000, -nan
0.00833333333333, 119, 1.00000000000000, 0.00089092653965
0.00500000000000, 199, 2.00000000000000, 0.00104751184602
0.00625000000000, 159, 1.00000000000000, 0.00054349407650
0.00416666666667, 239, 2.00000000000000, 0.00072840582581
0.00500000000000, 199, 1.00000000000000, 0.00031346384558
0.00416666666667, 239, 1.00000000000000, 0.00022695391384
